oracle-plugin/src/e2e-test/java/io.cdap.plugin/utils/E2ETestConstants.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.utils;
+
+/**
+ * E2E test constants.
+ */
+public class E2ETestConstants {
+  public static final String ERROR_MSG_INVALID_REFERENCE_NAME = "errorMessageOracleInvalidReferenceName";
+  public static final String ERROR_MSG_INVALID_TABLE_NAME = "errorMessageInvalidTableName";
```

Comment:
we will not be needing this class if we use common framework steps.

Reply:
Done.